src/internal/connector/data_collections.go

```diff
@@ -95,6 +95,7 @@ func (gc *GraphConnector) DataCollections(
 			ctx,
 			gc.itemClient,
 			sels,
+			metadata,
```

Comment:
this is for delta token incrementals. We should leave it out for now so we don't get unexpected behavior
```suggestion
			nil,
```

Reply:
Leaving the parameter in the call. Disabled within DataCollections() function See [05a83de](https://github.com/alcionai/corso/pull/2528/commits/05a83de91a67c9bcac79a1db1c017a1e5611d20d)

Reply:
👍 thanks for the clarification